src/helpers/projectHelpers.ts

```diff
@@ -343,8 +358,16 @@ export const reconcileProject = async (
   const web3 = getWeb3(chain);
   const contract = getContract(web3, abis[project_id], contract_address);
 
+  const client = getViemClient(chain);
+  const contractUsingViem = getContractUsingViem({
+    address: contract_address as `0x${string}`,
```

Comment:
I like how they use `0x${string}`, we can add this throughout at some point.

Reply:
Yeah it's a cool piece of validation. Didn't know you could use template strings as types

Reply:
Yep you can use any specific piece of data as a type. That's what 'as const' does.